tests/src/main/java/com/hazelcast/stabilizer/tests/map/helpers/IntegerStringEntryListener.java

```diff
@@ -67,31 +67,11 @@ public void mapCleared(MapEvent mapEvent) {
     }
 
     private void delay() {
-        if (maxDelayMs != 0) {
+        if (minDelayMs >= 0 && maxDelayMs > 0) {
```

Comment:
so changing this is a bit pedantic.  and I will also now say if its 0,  don't we get a problem ?